Build failed on RPi

## Description
<!--
pi@raspberrypi:~/Rust/azul_test $ cargo run
   Compiling azul v0.1.0 (https://github.com/maps4print/azul#bb5ab4c2)
error[E0308]: mismatched types
  --> /home/pi/.cargo/git/checkouts/azul-f3b1e21a5e685cad/bb5ab4c/cargo/azul/../../azul/text_shaping.rs:87:28
   |
87 |             hb_blob_create(font_ptr, font_bytes.len() as u32, MEMORY_MODE_READONLY, user_data_ptr, destroy_func)
   |                            ^^^^^^^^ expected u8, found i8
   |
   = note: expected type `*const u8`
              found type `*const i8`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: Could not compile `azul`.

-->

## Version / OS
<!--
Raspbian Buster
-->

* azul version:
0.1.0

* Operating system:
Linux RPi

* Windowing system (X11 or Wayland, Linux only):
X11

## Steps to Reproduce
<!--
cargo build
-->


## Additional Information
<!--
Built ok on Windows and Linux Mint.
-->

--- a/azul/text_shaping.rs
+++ b/azul/text_shaping.rs
@@ -82,7 +82,7 @@ impl<'a> HbFont<'a> {
         let user_data_ptr = ptr::null_mut();
         let destroy_func = None;
 
-        let font_ptr = font_bytes.as_ptr() as *const i8;
+        let font_ptr = font_bytes.as_ptr() as *const c_char;
         let hb_face_bytes = unsafe {
             hb_blob_create(font_ptr, font_bytes.len() as u32, MEMORY_MODE_READONLY, user_data_ptr, destroy_func)
         };

